### PR TITLE
Better stats

### PR DESCRIPTION
**Pull request by ramshorst**

### CODE REVIEW

File: less/base/_stats.less [hunk 1]
```diff
@@ -24,6 +25,21 @@
     line-height: 6rem;
     margin-top: 8px;
   }
+
+  .upf-stat__data--null {
+    color: darken(@upf-gray-light, 5%);
```
Comment (phndiaye) [inline less/base/_stats.less]: no way to use the text colors ?

File: less/core/_colors.less [hunk 2]
```diff
@@ -41,7 +41,7 @@
 // --------------------------------------------------
 
 @upf-feedback-success: #22B473;
-@upf-feedback-error: @upf-primary-orange;
+@upf-feedback-error: #FF0000;
```
Comment (phndiaye) [inline less/core/_colors.less]: again with the feedback error color changes ? 😛

File: less/base/_stats.less (hunk 1 above)
Comment (ramshorst) [reply less/base/_stats.less]: Not that I could find. Can you find one that matches ? Everywhere i found this color it was done like this :S

Comment (phndiaye) [reply less/base/_stats.less]: what about this one: https://github.com/upfluence/oss/blob/master/less/core/_variables.less#L22 , which is also used for the label

Comment (ramshorst) [reply less/base/_stats.less]: I tryed that one; it is to dark. The same for @color-text-lighter :(

**Comment by phndiaye** [comment]
LGTM 👍

**Pull request merged**